packages/blade/src/components/Input/_decisions/decisions.md

```diff
@@ -0,0 +1,212 @@
+| fullWidth                                                                                          | `boolean`                         | No                                                                                  | `false`                                                   | **Discuss if we want to add support for full-width input fields.**                                                                                                                                                                                                                    |
+| autofillSuggestionType                                                                             | `password`, `passwordNew`, `none` | No                                                                                  |                                                           | Specifies autocomplete hints for the system, so it can provide autofill                                                                                                                                                                                                               |
+| keyboardReturnKeyType. [Check this table for full reference](#return-key-types-for-web-and-native) | `go`, `done`, `next`, `send`      | No                                                                                  | Closest based on the `type` attribute                     | Determines how the return key should look on the keyboard on mobile devices or virtual keyboard                                                                                                                                                                                       |
+
+## Notes
+
+Reusable components candidate
+
+- label
+- helpText/errorText/successText
+- character counter(only input field)
+- clear icon button
+- prefix/suffix
+
+## Open questions
+
+- [ ] Do we have a usecase of using textfield without label or form level things?
+- [ ] Do we need to expose `autoCapitalize` for native?
+- [ ] Do we need to expose `onSubmitEditing` for native?
+- [ ] Do we need to fix the height for reserving space for help/error/success text?
+- [ ] motion for the textfield animation in active state
+- [ ] fullWidth inputField do we have a use case?
+- [ ] textContentType for autocomplete on native ios only
+  - https://reactnative.dev/docs/textinput#textcontenttype-ios
+- [ ] what is the default position for loader in loading state? following are some options
```

Comment:
Generally should be towards the right end of the input so as to not obscure the text that could be in the input field itself. If a clear button is present it could replace the clear button and show the loading state (as there is no usage of the clear button during the loading state). Putting it on the left will cause the text to move to the right when the loader appears if we decide to have the text during loading.